forc-util/src/lib.rs

```diff
@@ -561,10 +561,7 @@ fn format_diagnostic(diagnostic: &Diagnostic) {
         label.clear();
         if diagnostics.reason().is_some() {
             label.push_str(diagnostics.reason().unwrap().description());
```

Comment:
why not `if let Some(reason)` here?

Reply:
Well, I guess because the line was originally written while I was still catching up with Rust idioms :-) Thanks @xunilrj  the suggestion is applied! Can you please re-approve the PR?